app/src/androidTest/java/com/android/sample/screen/CoachFeedBackScreenTest.kt

```diff
@@ -49,7 +50,7 @@ class CoachFeedBackScreenTest {
     composeTestRule.onNodeWithTag("animatedText").performScrollTo().assertIsDisplayed()
     composeTestRule.onNodeWithTag("rankCircle").performScrollTo().assertIsDisplayed()
     composeTestRule
-        .onNodeWithTag("rankText")
+        .onNodeWithTag("rankText", useUnmergedTree = true)
```

Comment:
Whats the point of using useUnmergedTree here ?

Reply:
Tests wouldn't pass without it, but it did before, I'm not sure why but this fixed the issue